Formula/browserpass.rb

```diff
@@ -0,0 +1,54 @@
+class Browserpass < Formula
+  desc "Native component for Chrome & Firefox password management add-on"
+  homepage "https://github.com/dannyvankooten/browserpass"
+  url "https://github.com/dannyvankooten/browserpass/archive/2.0.8.tar.gz"
+  sha256 "1747fc24e51128811eb679b43ab23a8cce6ac566620112f0ca9fa129ace4a928"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+  depends_on "node" => :build
+  depends_on "yarn" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/dannyvankooten/browserpass").install buildpath.children
+    cd "src/github.com/dannyvankooten/browserpass" do
+      system "make", "deps"
+      system "make", "js"
+      system "make", "browserpass-darwinx64"
+      mkdir "out"
+      mkdir "out/bin"
+      mkdir "out/share"
+      cp "browserpass-darwinx64", "out/bin/browserpass"
+      cp "install.sh", "out/bin/browserpass-setup"
+      cp "firefox/host.json", "out/share/firefox-host.json"
+      cp "chrome/host.json", "out/share/chrome-host.json"
+      cp "chrome/policy.json", "out/share/chrome-policy.json"
+      dir = csh_quote(HOMEBREW_PREFIX)
+      inreplace "out/bin/browserpass-setup", /^(BIN_DIR=).*$/, "\\1\"#{dir}/bin\""
+      inreplace "out/bin/browserpass-setup", /^(JSON_DIR=).*$/, "\\1\"#{dir}/share/browserpass\""
+      bin.install Dir["out/bin/*"]
+      pkgshare.install Dir["out/share/*"]
+      ohai "#{Tty.magenta}** To complete installation of browserpass, do the following:#{Tty.reset}"
+      puts "(1) Install the browserpass-ce add-on in your browser."
+      puts "    - Chrome: https://chrome.google.com/webstore/detail/browserpass-ce/naepdomgkenhinolocfifgehidddafch"
```

Comment:
`install.sh` places the `chrome-policy.json`, which in turn forces Chrome to automatically download the browser extension. However I noticed that on linux this works only if you run `install.sh` as root. 

No action needed, just wanted to let you know this, maybe check how `install.sh` behaves on Mac, if it automatically installs Chrome extension you can reduce the manual steps.

Reply:
Doesn't seem to happen in Chrome. Are there any docs about this behavior?

Reply:
The feature is called [ExtensionInstallForcelist](https://dev.chromium.org/administrators/policy-list-3#ExtensionInstallForcelist), I think it is usually used by companies to enforce installation of a certain extension. When I have the `chrome-policy.json` stored in the following location on Linux, Chrome automatically installs the extension:

`/etc/opt/chrome/policies/managed/com.dannyvankooten.browserpass.json`

![image](https://user-images.githubusercontent.com/1177900/33232790-8cfc5f22-d20c-11e7-8b15-ec0144f00cb8.png)